ddp-workspace/projects/ddp-sdk/src/lib/models/statistic.ts

```diff
@@ -0,0 +1,14 @@
+export interface Statistic {
+  configuration: {
+    type: string;
+    questionStableId: string | null;
+    answerValue: string | null;
+  };
+  statistics: {
+    name: string;
+    data: {
+      count: string;
+      optionDetails?: any;
```

Comment:
We can use `ActivityPicklistOption` interface instead of any.

Reply:
Fixed.